packages/core/src/expression-builders/col.ts

```diff
@@ -0,0 +1,29 @@
+import { BaseSqlExpression } from './base-sql-expression.js';
+
+/**
+ * Do not use me directly. Use {@link col}
+ */
+export class Col extends BaseSqlExpression {
+  private readonly col: string[] | string;
+
+  constructor(identifiers: string[] | string, ...args: string[]) {
+    super();
+    // TODO(ephys): this does not look right. First parameter is ignored if a second parameter is provided.
```

Comment:
I think we'll need to write some unit tests on how people might use it now and see what the best type for it would be. But it indeed looks off

Reply:
From what I researched:

- It was a bug introduced _years_ ago when migrating everything to ES6 classes
- No one ever noticed it, or at least no one ever reported this
- This multi-parameter usage is not mentioned anywhere in our documentation

I think it's safe to say that no one ever used this multi-column implementation, but it's still preserved in the other PR